tcms/kiwi_auth/test_admin.py

```diff
@@ -109,3 +111,51 @@ def test_admin_can_add_new_users(self):
 
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertTrue(get_user_model().objects.filter(username='added-by-admin').exists())
+
+
+class TestGroupAdmin(LoggedInTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.tester.is_superuser = True
+        cls.tester.save()
+
+        cls.group = GroupFactory(name='NewGroupName')
+        cls.group.save()
+        cls.defaultGroups = Group.objects.filter(name__in=['Administrator', 'Tester'])
+
+    def test_should_not_be_allowed_to_change_default_groups_names(self):
+        for group in self.defaultGroups:
+            response = self.client.get(reverse('admin:auth_group_change', args=[group.id]))
+            self.assertNotContains(response,
+                                   '<input type="text" name="name" value="%s" class="vTextField"'
+                                   ' maxlength="150" required="" id="id_name">'
+                                   % group.name)
+            self.assertContains(response, '<div class="grp-readonly">%s</div>' % group.name)
+
+    def test_should_not_be_allowed_to_delete_default_groups_names(self):
+        for group in self.defaultGroups:
+            response = self.client.get(reverse('admin:auth_group_change', args=[group.id]))
+            self.assertNotContains(response, '<a href="%s" class="grp-button grp-delete-link">Delete</a>'
+                                   % reverse('admin:auth_group_delete', args=[self.group.id]))
+
+    def test_should_be_allowed_to_create_new_group(self):
+        response = self.client.get(reverse('admin:auth_group_add'))
+        self.assertContains(response, '<h1>Add group</h1>')
+        self.assertContains(response,
+                            '<input type="text" name="name" class="vTextField" maxlength="150" required id="id_name">')
+
+    def test_should_be_able_to_delete_a_group(self):
+        response = self.client.get(reverse('admin:auth_group_delete', args=[self.group.id]), follow=True)
+        self.assertEqual(HTTPStatus.OK, response.status_code)
+        self.assertContains(response, 'Are you sure you want to delete the group "%s"?' % self.group.name)
```

Comment:
This text should be translated.

There is also a second sentence to this and it is probably 1 entire string. OTOH there is a much simpler "Are you sure ?" above all of this.